suite-common/wallet-utils/src/settingsUtils.ts

```diff
@@ -1,9 +1,19 @@
 import { PROTO } from '@trezor/connect';
+import { networksCompatibility as NETWORKS, NetworkSymbol } from '@suite-common/wallet-config';
 
 export const getLocalCurrency = (localCurrency: string) => ({
     value: localCurrency,
     label: localCurrency.toUpperCase(),
 });
 
-export const getAreSatoshisUsed = (bitcoinAmountUnit: PROTO.AmountUnit) =>
-    bitcoinAmountUnit === PROTO.AmountUnit.SATOSHI;
+export const getAreSatoshisUsed = (bitcoinAmountUnit: PROTO.AmountUnit, symbol: NetworkSymbol) => {
+    const areSatsDisplayed = bitcoinAmountUnit === PROTO.AmountUnit.SATOSHI;
+
+    const areUnitsSupportedByNetwork =
+        symbol &&
+        NETWORKS.find(({ symbol: networkSymbol }) => networkSymbol === symbol)?.features?.includes(
+            'amount-unit',
+        );
+
+    return areSatsDisplayed && areUnitsSupportedByNetwork;
```

Comment:
check for device support is not relevant here? https://github.com/trezor/trezor-suite/blob/fix/send-toast-units/packages/suite/src/hooks/wallet/useBitcoinAmountUnit.ts/#L33

Reply:
Nope, that one is for knowing if it's possible to send. I actually wanted to rename the `areSatsUsed` flag to `areSatsSendable` since it's only used in that context.

Reply:
f78be0db9acc6428e7c0442fe4bfed17369f5b24